module/zfs/vdev_label.c

```diff
@@ -208,6 +208,40 @@ vdev_label_write(zio_t *zio, vdev_t *vd, int l, abd_t *buf, uint64_t offset,
 	    ZIO_PRIORITY_SYNC_WRITE, flags, B_TRUE));
 }
 
+void
+vdev_mmp_read(zio_t *zio, vdev_t *vd, int l, int n, abd_t *buf,
```

Comment:
Better yet, let's simply remove this function which appears to have been added for completeness but is entirely unused.